cms/djangoapps/contentstore/tests/test_course_settings.py

```diff
@@ -1423,30 +1423,6 @@ def test_validate_update_set_proctortrack_provider_with_valid_escalation_email(s
         self.assertIn('proctoring_provider', test_model)
         self.assertIn('proctoring_escalation_email', test_model)
 
-    @override_settings(
-        PROCTORING_BACKENDS={
```

Comment:
While it's nice to remove no longer used unit tests, but I believe you should add unit test for the added warning messages on course settings.